TYPO3.Flow/Classes/TYPO3/Flow/Cache/CacheManager.php

```diff
@@ -358,7 +358,7 @@ protected function flushConfigurationCachesByChangedFiles(array $changedFiles)
         $objectConfigurationCache = $this->getCache('Flow_Object_Configuration');
         $caches = array(
             '/Policy\.yaml/' => array('Flow_Security_Authorization_Privilege_Method', 'Flow_Persistence_Doctrine', 'Flow_Persistence_Doctrine_Results', 'Flow_Aop_RuntimeExpressions'),
-            '/Routes([^\/]*)\.yaml/' => array('Flow_Mvc_Routing_Route', 'Flow_Mvc_Routing_Resolve'),
+            '/Routes|Settings([^\/]*)\.yaml/' => array('Flow_Mvc_Routing_Route', 'Flow_Mvc_Routing_Resolve'),
```

Comment:
@albe that's probably difficult to implement, as the file monitor only tells us "which files have changes" and not "what has changed in them"...

@bwaidelich to me, this cache flushing is not a big problem; as it won't occur in production.

However, a comment would be really good at this point explaining why Routing also depends on Settings.yaml. This would be at least a question I'd have when I just see this code standalone.